plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/PrivilegeComponent.groovy

```diff
@@ -52,6 +53,7 @@ import javax.validation.groups.Default
 @Named
 @Singleton
 @DirectAction(action = 'coreui_Privilege')
+@CompileStatic
```

Comment:
example of @CompileStatic on this class, chosen because it required far less change than others; RepositoryComponent for instance reported a boatload of compile errors with this applied